tests/WorkOS/MFATest.php

```diff
@@ -156,6 +156,49 @@ public function testVerifyChallenge()
         $this->assertSame($verifyChallengeResponseFixture, $verifyChallenge->toArray());
     }
 
+    public function testGetFactor()
+    {
+        $authenticationFactorId = "auth_factor_01FXNWW32G7F3MG8MYK5D1HJJM";
+        $getFactorPath = "auth/factors/${authenticationFactorId}";
+
+        $result = $this->getFactorResponseFixture();
+
+        $this->mockRequest(
+            Client::METHOD_GET,
+            $getFactorPath,
+            null,
+            null,
+            true,
+            $result
+        );
+
+        $factor = $this->mfa->getFactor($authenticationFactorId);
+        $factorFixture = $this->getfactorFixture();
+
+        $this->assertSame($factorFixture, $factor->toArray());
+    }
+
+    public function testDeleteFactor()
+    {
+        $authenticationFactorId = "auth_factor_01FXNWW32G7F3MG8MYK5D1HJJM";
+        $deleteFactorPath = "auth/factors/${authenticationFactorId}";
+        $responseCode = 204;
+
+        $this->mockRequest(
+            Client::METHOD_DELETE,
+            $deleteFactorPath,
+            null,
+            null,
+            true,
+            null,
+            null,
+            $responseCode
+        );
+
+        $response = $this->mfa->deleteFactor($authenticationFactorId);
+        $this->assertSame(204, $responseCode);
```

Comment:
I think we'll want to change $responseCode to $response, so we aren't just testing to see if a 204 matches a variable explicitly set as '204' on line 185. We actually want to make sure we call a deleteFactor() in the test.